### ISSUE TITLE
Create a home/features Page

### ISSUE THREAD

**Issue by odysseywestra** [mypaint/mypaint.github.io#15]
We need to create a home page that highlights some of the key features of MyPaint. The blog/new itself will most likely be on its own page, but I would like to have a recent news area in the page. Some of the Features I know are: 

* Customizable Brush Engine that emulates not only Traditional brushes but also more exotic ones.
* True Infinite Canvas.
* Graphics Tablet Support.
* No-Distraction Full screen UI
* Layers with group folders,vector layers and external application support.
* Cross-Platform

Like I said where looking for key features that make MyPaint stand out from the crowd. Ones that stand out to me are its brush engine, layers features, and No-distraction UI. Right now we just need information. We can worry about design later. If you need help creating a good flow explain x key feature, take a look at http://www.hemingwayapp.com/ which @achadwick mentioned in #7 .

Let me know your guys ideas down below.

**Comment by achadwick** [comment]
Should this be where the screenshots we've been gathering in mypaint/mypaint#149 go, or does that deserve its own page?

Screenshots are one of the things people look for when first confronted with a new program. If some can be incorporated onto the home page as feature previews, that would look great!

**Comment by odysseywestra** [comment]
Yeah, that great suggestion actually. We don't have to transfer them over here since it is linked now. We could close that issue and point them here to post their screenshots too. I'll edit the first post accordingly.

**Comment by odysseywestra** [comment]
@achadwick Opened a Discussion Thread about needing screenshots on our Community Fourms. You can post them there as well. 

http://community.mypaint.org/t/screenshots-of-1-2-0-during-development/44?u=odysseywestra

---
Update: Sorry I posted that in the wrong thread.

**Comment by ilovepumpkin** [comment]
![vera_2015_birthday jpg - mypaint](https://cloud.githubusercontent.com/assets/11310645/11766805/f14ca10c-a1d1-11e5-8276-84d2420b08c2.png)

**Comment by achadwick** [comment]
@ilovepumpkin That screenshot looks great! There's such a lovely human warmth and glow to it ^^. Are you wanting us to use it on our website or for other promotional purposes?

You'd need to allow us (and others) to reuse it for that: here's our [licensing policy](https://github.com/mypaint/mypaint/wiki/Licensing-policy#supplemental-artwork-or-promotional-material). I would suggest putting a subtle but unambiguous note on it saying "© 2015 [YOUR NAME HERE], CC BY-SA 3.0".

You should be able to edit existing posts and replace images. Just drop another comment to state your intent if you do that.

**Comment by ilovepumpkin** [comment]
@achadwick , I have updated my screenshot by editing my original one a few days ago ( right after I saw your reply), but unfortunately I forgot to start this new comment (you are only notified when there is a new comment, right? - I guess so), :).
